docs/i18n/zh_CN/benchmarks/spot5/README.md

```
@@ -0,0 +1,328 @@
+
+在本仓库中，每个发布的实例都作为独立的 benchmark 测试实例存储：
+
+```text
+benchmarks/spot5/dataset/cases/<split>/<case_id>/<case_id>.spot
+```
+
+## 实例文件格式（.spot）
+
+### 整体结构
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)